stats_go.go

```diff
@@ -61,3 +61,18 @@ func (r StatsReport) GetICECandidatePairStats(c *ICECandidatePair) (ICECandidate
 	}
 	return candidateStats, true
 }
+
+// GetCodecStats is a helper method to return the associated stats for a given Codec
+func (r StatsReport) GetCodecStats(c *RTPCodec) (CodecStats, bool) {
```

Comment:
`GetMediaStats` is probably better. I need to look at `pc-stats` and see if a better name comes out.